mava/components/building/environments.py

```diff
@@ -68,6 +68,10 @@ def on_building_init_start(self, builder: SystemBuilder) -> None:
         )
         builder.store.extras_spec = {}
 
+        builder.store.obs_normalisation_start = (
+            self.config.environment_factory().obs_normalisation_start_index
+        )
+
```

Comment:
We call the env factory above, rather create the env once for both of these calls:

```
env = self.config.environment_factory()
...
MAEnvironmentSpec(env)
...
env.obs_normalisation_start_index
```

Reply:
Cool will do that

Reply:
Done